source/fundamentals/data-formats/document-data-format-record.txt

```diff
@@ -191,3 +190,38 @@ as shown in the following code:
 
       NetworkDeviceRecord[deviceId=fedc..., name=Enterprise Wi-fi, deviceType=router]
 
+Operations with Recursive Types
+-------------------------------
+
+The driver natively supports encoding and decoding cyclical
+references without causing runtime recursion. The following code
```

Comment:
It's not the references that are cyclical.  The driver doesn't actually handle that.  It's when there are cycles in the type definitions.  In this case, RecordTree has two components of type RecordTree.  So there is a (short) cycle in the type definitions.  But the support generalizes to any sort of cycle in the type system, e.g.  RecordOne with a component of type RecordTwo with a component of type RecordThree with a component of type RecordOne.

Reply:
do you think that I should expand the example to add a second (or third) record definition to fully illustrate support for cycles? I will also reword.

Reply:
Probably not necessary, I was just trying to explain it to you.  But I don't think we should use the phrase "cyclical references" as that will most likely be interpreted as references of instances, not types.